product_docs/docs/epas/16/epas_deprecates/index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Deprecates"
+description: "Provides a list of deprecated features, extensions and parameters for upgrading or migrating to EDB Postgres Advanced Server 16.x. "
+---
+
+## Deprecated Features
+Below features are deprecated from this release, please check and plan for migrating to the alternative feature.
+
+|         Feature          | Description |                      Alternative                       |
+| ------------------------ | ------------ | ---------------------------------------------------------- |
+| Index Advisor | The Index Advisor utility helps determine the columns to index to improve performance in a given workload.  |  Index Advisor features are now included in the [EDB Query Advisor](/pg_extensions/query_advisor/).
+
+## Deprecated Extenstions
+Currently there is no deprecated extension in this release.
+
+## Deprecated Parameters
+Below parameters are deprecated from this release, please chacke and plan for migrating to the alternative parameter.
+
+|         Feature          | Description |                      Alternative                       |
+| ------------------------ | ------------ | ---------------------------------------------------------- |
+| edb_enable_pruning    | fast pruning for EDB-partitioned tables | None                          |
```

Comment:
Hi team,
Greetings. Hope you are doing well and healthy.

The comment you point out is exactly what I'm not sure. By definition, if a parameter is not discarded, but deprecated, it should not be on the release note, because even it is not suggested any longer, you still can use it in this release, right?

Back to "edb_enable_pruning", it is said to be "deprecated" since version 16, so it is expected that we can still use it in v16, but actually when I diff the v15 and v16 parameters, I found that this parameter is completely discarded from the database. So then come the question, shall we see this as deprecated feature or discard feature.
If this is a deprecated feature, the "Alternative" should be, "please thinking of turning enable_partition_pruning to on instead", but if it is discarded feature, I think we should rather edit release note.

Link:
https://www.postgresql.jp/document/16/html/runtime-config-query.html#GUC-ENABLE-PARTITION-PRUNING

Reply:
Thanks @yuki-tei, I am going to let a colleague handle this, as she is in close contact with the EPAS team. Will update you when we have more information.

Reply:
Thanks @gvasquezvargas. Please feel free to take time checking with your internal team. Apologies for that I may haven't responded to the navigation issue, I agree that this page is proper to place as below.

navigation: 
  - [...]
  - upgrading
  - epas_deprecates
  - database_administration
  - [...]

Reply:
@tusharahuja, can you please look into this and give your suggestions here?

Reply:
@yuki-tei, this note is already added to the EPAS 16.1 release notes.Link for your reference: https://www.enterprisedb.com/docs/epas/16/epas_rel_notes/epas16_rel_notes/#:~:text=Deprecation,Query%20Advisor.